beamtech/.github
# Add techdocs

## Context

This PR does the following:
- Adds techdocs generation CI step
- Adds techdocs annotation and mkdocs config for the repo

## Validation

1. Checkout the `main` branch of the `backstage` repo
2. `yarn install`
3. Add this block to `app-config.yaml#catalog.locations`:

   ```
    - type: github-discovery
      target: https://github.com/beamtech/.github/blob/add-techdocs/**/catalog-info.yaml
   ```

4. `yarn dev`
5. Ensure that the techdocs show up for the `github` component and its subcomponents

#### [How to QA](https://beamdental.atlassian.net/wiki/spaces/ENG/pages/230293509/How+to+QA)
